packages/backend/src/tenant/service.ts

```diff
@@ -85,16 +84,13 @@ async function createTenant(
   return deps.knex.transaction(async (trx) => {
     let tenant: Tenant
     try {
-      // create tenant on backend
-      tenant = await Tenant.query(trx).insert({
-        kratosIdentityId: uuidv4()
-      })
+      const tenantData = {
+        name: options.name,
+        kratosIdentityId: uuidv4(),
+        endpoints: options.endpoints
```

Comment:
this allows Objection to automagically create the endpoints in the tenantsEndpoints model, correct?

Reply:
correct